File: Practica_7/Parte_II/Parte2.2/programa.java
import java.lang.Math;

public class programa {

	public static void main(String[] args) {

        int seno=0,coseno=0,tangente=0,logaritmo=0,raiz=0;
        double i = 0;	

        while(i < 1000000){
            seno += Math.sin(i);
            coseno += Math.cos(i);
            tangente += Math.tan(i);
            logaritmo += Math.log(i);
            raiz += Math.sqrt(i); 
            i++;
        }
		System.out.println("termine");
	}

}

File: Parcial_1/Practica_7/Parte_II/Parte2.2/programa.cpp
#include<iostream>
#include <math.h>
#define MAX 50000000

using namespace std;

int main (){

    long seno,coseno,tangente,logaritmo,raiz;
    double i = 0;

    while(i < MAX){
        seno += sin(i);
        coseno += cos(i);
        tangente += tan(i);
        logaritmo += log(i);
        raiz += sqrt(i);
        i++;
    }
    cout << "termine" << endl;
    return 0;
}